Code/client/Services/Debug/Views/EntitiesView.cpp

```diff
@@ -170,6 +170,16 @@ void DebugService::DisplayFormComponent(FormIdComponent& aFormComponent) const n
     ImGui::InputScalar("Is dead?", ImGuiDataType_U8, &isDead, 0, 0, "%" PRIx8, ImGuiInputTextFlags_ReadOnly);
     int isRemote = int(pActor->GetExtension()->IsRemote());
     ImGui::InputScalar("Is remote?", ImGuiDataType_U8, &isRemote, 0, 0, "%" PRIx8, ImGuiInputTextFlags_ReadOnly);
+    auto owner = pActor->GetCommandingActor();
+    int commandingActorId = int(owner ? owner->formID : 0x0);
+    if (ImGui::Button("Take Commmand"))
+    {
+        pActor->SetCommandingActor(m_world.playerHandler);
```

Comment:
Don't call in game functions that change state, like setters, from the ImGui thread. It's not thread safe. I understand that this is test code and will be deleted, but just letting you know for the future.

Reply:
good to know, thanks!

Reply:
has now been removed